node/resolvers/deleteIndex.ts

```diff
@@ -0,0 +1,46 @@
+import { EXTENDED_INDEX_FILE, getBucket, hashString } from '../utils'
```

Comment:
I've noticed that this file is almost the same as the `saveIndex` one, do you think we could refactor it to reuse code? Like have a function to get `extendedIndexes` and another to save, so in each resolver, we would only call them and do what is different - for example, filter in the delete case.

Reply:
@lariciamota Done! I also updated my workspace link with these changes.